[2023-06-28T22:36:30.676+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-28T18:35:24-04:00 [queued]>
[2023-06-28T22:36:30.687+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-28T18:35:24-04:00 [queued]>
[2023-06-28T22:36:30.687+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-28T22:36:30.687+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-28T22:36:30.687+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-28T22:36:30.731+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): transfer_s3_to_rds> on 2023-06-28 22:35:24+00:00
[2023-06-28T22:36:30.752+0000] {standard_task_runner.py:55} INFO - Started process 7731 to run task
[2023-06-28T22:36:30.759+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_pnadc_data_new', 'transfer_s3_to_rds', 'manual__2023-06-28T18:35:24-04:00', '--job-id', '309', '--raw', '--subdir', 'DAGS_FOLDER/get_pnadc_data_new.py', '--cfg-path', '/tmp/tmp7_wyc3nr']
[2023-06-28T22:36:30.762+0000] {standard_task_runner.py:83} INFO - Job 309: Subtask transfer_s3_to_rds
[2023-06-28T22:36:30.869+0000] {task_command.py:389} INFO - Running <TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-28T18:35:24-04:00 [running]> on host d9b032dfc97f
[2023-06-28T22:36:31.246+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_pnadc_data_new
AIRFLOW_CTX_TASK_ID=transfer_s3_to_rds
AIRFLOW_CTX_EXECUTION_DATE=2023-06-28T22:35:24+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-28T18:35:24-04:00
[2023-06-28T22:36:31.279+0000] {base.py:73} INFO - Using connection ID 'aws_conn' for task execution.
[2023-06-28T22:36:31.280+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_conn', conn_type='aws') credentials retrieved from login and password.
[2023-06-28T22:36:34.056+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-28T22:36:52.778+0000] {logging_mixin.py:137} INFO - End to add rows! 10000 rows added.
[2023-06-28T22:36:52.779+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-28T22:36:52.792+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=get_pnadc_data_new, task_id=transfer_s3_to_rds, execution_date=20230628T223524, start_date=20230628T223630, end_date=20230628T223652
[2023-06-28T22:36:52.880+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-28T22:36:52.930+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
